### PR TITLE
BER-5: Created unit test case demo

### PR DESCRIPTION
**Pull request by Sonaldev3**
- Created a demo for unit test
- Added a command for running all the test cases before running the build.
- Added a command for prod build without running the test cases.

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -338,6 +338,8 @@
 
   </div>
```
Comment (prashant-pareek) [inline src/app/app.component.html:339]: can we remove rest of the code that comes with angular setup?

File: src/app/core/components/unit-test-demo/unit-test-demo.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should be changed', function () {
+    const fixture = TestBed.createComponent(UnitTestDemoComponent);
+    const app = fixture.componentInstance;
+    // expect(app.heading).toBe('Angular Elements');
```
Comment (prashant-pareek) [inline src/app/core/components/unit-test-demo/unit-test-demo.component.spec.ts:29]: remove log

File: src/app/app.component.html (hunk 1 above)
Comment (Sonaldev3) [reply src/app/app.component.html:339]: Okay

File: src/app/core/components/unit-test-demo/unit-test-demo.component.spec.ts (hunk 2 above)
Comment (Sonaldev3) [reply src/app/core/components/unit-test-demo/unit-test-demo.component.spec.ts]: Removed

**Review by prashant-pareek — APPROVED** [review]

**Pull request merged**